src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -149,4 +149,16 @@ public void testOuterTypeNumber() throws Exception {
         testByConfig(
                 "OuterTypeNumber/context/defaultContextConfig.xml");
     }
+
+    @Test
+    @Ignore("until https://github.com/checkstyle/patch-filters/issues/139")
```

Comment:
Please explain in issue description why this test is ignored and/or what problem we have with this Check.

Reply:
https://github.com/checkstyle/patch-filters/pull/143#discussion_r460689358

Reply:
@baratali Ignore removed because #162 merged.